ddtrace/contrib/gevent/greenlet.py

```diff
@@ -3,6 +3,7 @@
 
 from .provider import CONTEXT_ATTR
 
+GEVENT_VERSION = tuple(map(int, gevent.__version__.split()[0].split('.')))
```

Comment:
There is also `gevent.version_info` which is a tuple.

Can likely just do:

```python
GEVENT_VERSION = gevent.version_info[0:3]
```

Reply:
https://github.com/gevent/gevent/blob/v1.1.0/gevent/__init__.py#L18

Reply:
Even better, thanks!